www/src/theme/styles/global.ts

```diff
@@ -20,11 +20,21 @@ const styles: Styles = {
     "::selection": {
       background: `selection`,
     },
-    "p + p": {
-      mt: 3,
+    "h2, h3, h4, h5, h6, p, ol, ul": {
+      mb: 5,
     },
-    "p + p ol, p + p ul": {
-      mt: -3,
+    "p+ol, p+ul": {
+      mt: -5,
+    },
+    "li > p": {
+      mb: 0,
+    },
+    "#content > * > p:nth-of-type(1)::first-letter": {
```

Comment:
Should look into factoring this out into something like prose.ts and only calling it for pages that have content.